packages/ui/src/components/w3m-spinner/styles.css

```diff
@@ -29,5 +29,12 @@ svg {
 svg circle {
   stroke-linecap: round;
   animation: dash 1.5s ease infinite;
+}
+
+.accent {
   stroke: var(--w3m-accent-color);
 }
+
+.white {
+  stroke: #fff;
```

Comment:
This doesn't really work in terms of light / dark modal, plus what if users override colors of a button or its text? Instead of `white` option here, maybe lets make it as "fill" (or whatever terminology is used for color of a text in a button) and use css variable that is used for button text color. This will make things more consistent.